frontend/src/main/scala/bloop/testing/LoggingEventHandler.scala

```diff
@@ -1,38 +1,40 @@
 package bloop.testing
 
-import bloop.logging.{DebugFilter, Logger}
+import bloop.logging.DebugFilter
+import bloop.logging.Logger
 import bloop.util.TimeFormat
 import ch.epfl.scala.bsp
 import ch.epfl.scala.bsp.BuildTargetIdentifier
-import ch.epfl.scala.bsp.endpoints.{Build, BuildTarget}
-import sbt.testing.{Event, Selector, Status}
+import ch.epfl.scala.bsp.endpoints.Build
+import ch.epfl.scala.bsp.endpoints.BuildTarget
+import ch.epfl.scala.debugadapter.DebuggeeListener
+import ch.epfl.scala.debugadapter.SingleTestResult
+import ch.epfl.scala.debugadapter.TestSuiteResult
+import ch.epfl.scala.debugadapter.TestResultEvent
+import ch.epfl.scala.debugadapter.util.TestSuiteEvent
+import ch.epfl.scala.debugadapter.{util => ch}
+import sbt.testing.Event
+import sbt.testing.Selector
+import sbt.testing.Status
+import sbt.testing.TestSelector
 
 import scala.collection.mutable
 import scala.meta.jsonrpc.JsonRpcClient
 import scala.util.Try
 
-sealed trait TestSuiteEvent
-object TestSuiteEvent {
-  case object Done extends TestSuiteEvent
-  case class Error(message: String) extends TestSuiteEvent
-  case class Warn(message: String) extends TestSuiteEvent
-  case class Info(message: String) extends TestSuiteEvent
-  case class Debug(message: String) extends TestSuiteEvent
-  case class Trace(throwable: Throwable) extends TestSuiteEvent
-
-  /** @param testSuite Class name of test suite */
-  case class Results(testSuite: String, events: List[Event]) extends TestSuiteEvent
-}
+import collection.JavaConverters._
 
-trait TestSuiteEventHandler {
-  def handle(testSuiteEvent: TestSuiteEvent): Unit
+trait TestSuiteEventHandler extends ch.TestSuiteEventHandler {
```

Comment:
```suggestion
trait BloopTestSuiteEventHandler extends ch.TestSuiteEventHandler {
```

we have similar convention for other classes coming from scala-debug-adapter